app/src/main/java/org/oppia/android/app/devoptions/markchapterscompleted/ChapterSummaryViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.android.app.devoptions.markchapterscompleted
+
+import org.oppia.android.app.model.ChapterPlayState
+import org.oppia.android.app.model.ChapterSummary
+
+/** [MarkChaptersCompletedItemViewModel] for displaying a chapter summary. */
+class ChapterSummaryViewModel(
+  val chapterIndex: Int,
+  val chapterSummary: ChapterSummary,
+  val nextStoryIndex: Int,
+  val storyId: String,
+  val topicId: String
+) : MarkChaptersCompletedItemViewModel() {
+  /**
+   * Function to check if a chapter is completed or not. It is used in
+   * [MarkChaptersCompletedFragmentPresenter] to decide if a chapter should be checked and disabled.
+   */
```

Comment:
Function KDocs should always read as a verb phrase. Consider this alternative:

```kotlin
/** Returns whether the chapter represented by the current view model is completed. */
```

Note the phrasing. Also, I omitted the "It is used" bit since that's unrelated to this class. We should only ever document what a function does and how it can be used, not how it's actually used. The latter is always changing & is better found by analyzing call sites rather than documentation.

Ditto elsewhere for all other KDocs.

Reply:
Done.